Spokestack/SpeechContext.swift

```diff
@@ -14,10 +14,12 @@ import Foundation
     @objc public var transcript: String = ""
     /// Current speech recognition confidence: [0-1)
     @objc public var confidence: Float = 0.0
-    /// Speech pipeline active indicator
-    @objc public var isStarted: Bool = false
     /// Speech recognition active indicator
     @objc public var isActive: Bool = false
-    /// Speech detected indicator
-    @objc public var isSpeech: Bool = false
+    /// Speech detected indicator. Default to true for non-vad activation
```

Comment:
this is used as part of the pipeline for vad and vad-listening components to adjust their state off of.

Reply:
Re-reading this, I should've been more specific. In case it wasn't clear, it was the default of `true` that I was objecting to, since I don't think `isSpeech` should be true unless there's a VAD telling you that speech has happened, or something's manually activated the pipeline.

Reply:
It was clear, i just wasn't considering the case of a manually-activated pipeline.